bin/llvm-kompile-clang

```diff
@@ -160,6 +165,11 @@ if [ "$main" != "python_ast" ]; then
       if ! $link; then
         modasm="$output_file"
       fi
+      if $visibility_hidden; then
+        modhidden="$tmpdir/hidden.bc"
+        run @OPT@ "$modopt" -load-pass-plugin "$LIBDIR/libSetVisibilityHidden.so" -set-visibility-hidden -o "$modhidden"
```

Comment:
I saw you had some issues on macOS that are now fixed; I wanted to confirm that the compiled pass library actually gets this name (rather than `.dylib`) on macOS.